tests/drivers/candev/main.c

```diff
@@ -226,24 +226,32 @@ if (IS_ACTIVE(CONFIG_USE_LOOPBACK_MODE)) {
         struct can_filter filter[4];
         filter[0].can_mask = 0x7FF;
         filter[0].can_id = 0x001;
-#if defined(MODULE_MCP2515)
-        filter[0].target_mailbox = 0;   /* messages with CAN ID 0x001 will be received in mailbox 0 */
+#if !defined(__linux__)
```

Comment:
Can `!defined(__linux__)` ever evaluate to False with `IS_ACTIVE(MCP2515_RECV_FILTER_EN)` evaluating to True?